content/tools/polyfill.md

```diff
@@ -27,16 +27,25 @@ summary: |
 
 {% import 'embed.macros.njk' as embed %}
 
-## Cascade Layers Polyfill
+## Popover Polyfill
```

Comment:
@SondraE I think we could consider adding a screenshot from the Popover (and also Layers?) demo pages, like we do for anchor positioning?

Reply:
Done